src/main/java/uk/gov/hmcts/divorce/systemupdate/schedule/SystemAlertApplicationNotReviewedTask.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.hmcts.divorce.systemupdate.schedule;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.sdk.type.YesOrNo;
+import uk.gov.hmcts.divorce.citizen.notification.JointApplicationOverdueNotification;
+import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
+import uk.gov.hmcts.divorce.systemupdate.service.CcdSearchCaseException;
+import uk.gov.hmcts.divorce.systemupdate.service.CcdSearchService;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.divorce.divorcecase.model.State.AwaitingApplicant2Response;
+
+@Component
+@Slf4j
+public class SystemAlertApplicationNotReviewedTask {
+
+    @Autowired
+    private CcdSearchService ccdSearchService;
+
+    @Autowired
+    private JointApplicationOverdueNotification jointApplicationOverdueNotification;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Scheduled(cron = "${schedule.joint_application_overdue}")
+    public void execute() {
+
+        log.info("Joint application overdue scheduled task started");
+
+        try {
+            final List<CaseDetails> casesInAwaitingApplicant2Response =
+                ccdSearchService.searchForAllCasesWithStateOf(AwaitingApplicant2Response);
+
+            for (final CaseDetails caseDetails : casesInAwaitingApplicant2Response) {
```

Comment:
this for loop is quite long, perhaps split the contents out into a private method?